src/DotNetEnv/Parsers.cs

```diff
@@ -187,7 +187,7 @@ private static string ToUtf32Char (string hex)
                     from partOfValue in NotControlNorWhitespace("$\"'") // quotes are not allowed in values, because in a shell they mean something different
                     select new ValueActual(string.Concat(inlineWhitespaces, partOfValue)))
                 .Many()
-                .Select(vs => new ValueCalculator(vs));
+                .Select(vs => new ValueCalculator(vs.ToArray()));
```

Comment:
Why is this getting added?

Reply:
Willbe solved with reverting that: https://github.com/tonerdo/dotnet-env/pull/93/files/e1e5f17d1f16eb4761136bffe777cade0240bdd5#r1510010054